src/components/Process/Create/Sidebar/CensusCreation.tsx

```diff
@@ -120,11 +112,33 @@ const GroupCensusCreation = () => {
     formState: { errors },
   } = useFormContext<Process>()
   const censusType = watch('censusType')
+  const { data: groups, fetchNextPage, hasNextPage, isFetching } = useGroups(6)
 
   return (
     <Box display='flex' flexDirection='column' gap={4}>
-      <GroupSelect />
-      <VoterAuthentication />
+      {groups?.length > 0 && (
+        <>
+          <GroupSelect
+            groups={groups}
+            fetchNextPage={fetchNextPage}
+            hasNextPage={hasNextPage}
+            isFetching={isFetching}
+          />
+          <VoterAuthentication />
+        </>
+      )}
+
+      {(!groups || groups?.length === 0) && (
+        <Text fontSize='xs' color='texts.subtle'>
+          <Trans i18nKey='process_create.census.group.no_groups'>
+            You don't have any groups yet.{' '}
+            <Link as={ReactRouterLink} to={Routes.dashboard.memberbase.base} textDecoration='underline'>
+              Create one here
+            </Link>{' '}
+            to start a process.
+          </Trans>
+        </Text>
```

Comment:
Let's change this to be a warning alert:

```suggestion
        <Alert status='warning' fontSize='xs' color='texts.subtle'>
          <AlertDescription>
            <Trans i18nKey='process_create.census.group.no_groups'>
              You don't have any groups yet.{' '}
              <Link as={ReactRouterLink} to={Routes.dashboard.memberbase.base} textDecoration='underline'>
                Create one here
              </Link>{' '}
              to start a process.
            </Trans>
          </AlertDescription>
        </Alert>
```